### PR TITLE
Add initial operations

### PR DESCRIPTION
**Pull request by AleksanderWWW**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: src/neptune/core/operations/operation.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+    @staticmethod
+    def from_dict(data: dict) -> "AssignFloat":
+        return AssignFloat(data["path"], data["value"])
+
+
+@dataclass
+class AssignInt(Operation):
+
+    value: int
```
Comment (szysad) [inline src/neptune/core/operations/operation.py:102]: some classes have extra space and some do not. Did you run formatter on all those files?

File: src/neptune/core/operations/operation.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+    def to_dict(
+        self,
+        value_serializer: Callable[[T], Any] = lambda x: x,
+    ) -> Dict[str, Union[T, Optional[float], float]]:
+        return {"value": value_serializer(self.value), "step": self.step, "ts": self.ts}
+
+    @staticmethod
+    def from_dict(data: Dict[str, Any], value_deserializer: Callable[[T], Any] = lambda x: x) -> "LogSeriesValue[T]":
+        return LogSeriesValue[T](value_deserializer(data["value"]), data.get("step", None), data["ts"])
```
Comment (szysad) [inline src/neptune/core/operations/operation.py:190]: `data.get("step", None)` is the same as `data.get("step")`

**Review by szysad — APPROVED** [review]
nice one!

File: src/neptune/core/operations/operation.py (hunk 2 above)
Comment (AleksanderWWW) [reply src/neptune/core/operations/operation.py:190]: good point, was just copy-pasting from `internal/operation.py`

File: src/neptune/core/operations/operation.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        cls._registry[cls.__name__] = cls
+
+    @abc.abstractmethod
+    def accept(self, visitor: "OperationVisitor[Ret]") -> Ret:
+        pass
+
+    def clean(self, operation_storage: OperationStorage) -> None:
+        pass
+
+    def to_dict(self) -> dict:
```
[the remark this answers is not in the document]
Comment (AleksanderWWW) [reply src/neptune/core/operations/operation.py:69]: sure

File: src/neptune/core/operations/operation.py (hunk 1 above)
Comment (szysad) [reply src/neptune/core/operations/operation.py]: I think we good rn

**Review by szysad — APPROVED** [review]
lgtm!

**Pull request merged**